src/golang/lib/job/config.go

```diff
@@ -40,6 +41,23 @@ type LambdaJobManagerConfig struct {
 	AwsSecretAccessKey string `yaml:"awsSecretAccessKey" json:"aws_secret_access_key"`
 }
 
+type DatabricksJobManagerConfig struct {
+	// WorkspaceURL is the full url for the Databricks workspace that
+	// Aqueduct operators will run on.
+	WorkspaceURL string `yaml:"workspaceUrl" json:"workspace_url"`
+	// AccessToken is a Databricks AccessToken for a workspace. Information on how
+	// to create tokens can be found here: https://docs.databricks.com/dev-tools/auth.html#personal-access-tokens-for-users
+	AccessToken string `yaml:"accessToken" json:"access_token"`
+	// Databricks needs an Instance Profile with S3 permissions in order to access metadata
+	// storage in S3. Information on how to create this can be found here:
+	// https://docs.databricks.com/aws/iam/instance-profile-tutorial.html
+	S3InstanceProfileARN string `yaml:"s3InstanceProfileArn" json:"s3_instance_profile_arn"`
```

Comment:
This should be fine for now, but I think we can have a task to look into merging this with the S3 configs?